NaZdrowie/src/components/organisms/useScreenNavigation.tsx

```diff
@@ -62,7 +62,7 @@ export const useScreensNavigation = () => {
     if (status === "granted") {
       navigate("QrScanner");
     } else {
-      // This needs to be replaced with our custom alert or sth
+      // Here action is required - so Alert stays
       Alert.alert(
         "Permission required",
```

Comment:
Shouldn't it be written in polish? We have polish app, and alert prompts in english

Reply:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa